[2024-01-20T20:00:00.355+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Kafka_Producer_2.coinMarket scheduled__2024-01-20T19:57:48.111996+00:00 [queued]>
[2024-01-20T20:00:00.361+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Kafka_Producer_2.coinMarket scheduled__2024-01-20T19:57:48.111996+00:00 [queued]>
[2024-01-20T20:00:00.362+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2024-01-20T20:00:00.374+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): coinMarket> on 2024-01-20 19:57:48.111996+00:00
[2024-01-20T20:00:00.378+0000] {standard_task_runner.py:57} INFO - Started process 352 to run task
[2024-01-20T20:00:00.381+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'Kafka_Producer_2', 'coinMarket', 'scheduled__2024-01-20T19:57:48.111996+00:00', '--job-id', '1219', '--raw', '--subdir', 'DAGS_FOLDER/kafka_stream_2.py', '--cfg-path', '/tmp/tmpw7w0g43r']
[2024-01-20T20:00:00.383+0000] {standard_task_runner.py:85} INFO - Job 1219: Subtask coinMarket
[2024-01-20T20:00:00.467+0000] {task_command.py:415} INFO - Running <TaskInstance: Kafka_Producer_2.coinMarket scheduled__2024-01-20T19:57:48.111996+00:00 [running]> on host 45c5bfab8aa1
[2024-01-20T20:00:00.540+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='me' AIRFLOW_CTX_DAG_ID='Kafka_Producer_2' AIRFLOW_CTX_TASK_ID='coinMarket' AIRFLOW_CTX_EXECUTION_DATE='2024-01-20T19:57:48.111996+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-20T19:57:48.111996+00:00'
[2024-01-20T20:00:10.582+0000] {logging_mixin.py:151} INFO - HTTPSConnectionPool(host='pro-api.coinmarketcap.com', port=443): Max retries exceeded with url: /v1/cryptocurrency/listings/latest?start=1&convert=USD (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f91ce940e20>: Failed to resolve 'pro-api.coinmarketcap.com' ([Errno -3] Temporary failure in name resolution)"))
[2024-01-20T20:00:10.583+0000] {python.py:194} INFO - Done. Returned value was: None
[2024-01-20T20:00:10.591+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=Kafka_Producer_2, task_id=coinMarket, execution_date=20240120T195748, start_date=20240120T200000, end_date=20240120T200010
[2024-01-20T20:00:10.609+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-01-20T20:00:10.626+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
